CHANGELOG.md

```diff
@@ -2,15 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## Unreleased
+## 1.0.2 (2020-06-18)
 
 ### ⚠ BREAKING CHANGES
-* **Tools:** Update Amplify tools script to resolve min CLI version for codegen changes ([#554](https://github.com/aws-amplify/amplify-ios/pull/554))
+* **DataStore/API** category using generated Model classes containing embedded types will require Amplify CLI 4.21.4 to generate working Model classes. This is to fix the bug in generating the incorrect selection set when using DataStore with sync enabled and API. The ModelFieldType `.customType` has been replaced with `embedded(type:)` and `embeddedCollection(of:)`. When upgrading to 1.0.2, please also upgrade Amplify CLI before running `amplify codegen models` to generate the new `Embeddable` types. We have considered this breaking change a non-major version bump as existing host applications will continue to experience the existing behavior after upgrading the library if they do not generate the new Model classes.
```

Comment:
Remove:
> We have considered this breaking change a non-major version bump as existing host applications will continue to experience the existing behavior after upgrading the library if they do not generate the new Model classes.

As discussed, this isn't a breaking change. To fix the bug update the CLI and the library. If you don't update the CLI and update the library, it doesn't break. There's a known incompatibility if someone updates the CLI and *not* the library, but that doesn't make this library release a breaking change.

Feel free to call out that updating the CLI must be accompanied by a library update, but again, not a breaking change in the lib

Reply:
my bad, gotcha. removing "We have considered..." note and improving wording

> Add Emeddable type to store schema info for custom types ([#539](https://github.com/aws-amplify/amplify-ios/pull/539)). This bug impacts developers with schemas containing embedded types when using DataStore with sync to cloud enabled or using API with Model classes. To fix this bug, upgrade both Amplify CLI to 4.21.4 and Amplify Libraries to 1.0.2. There is a known incompatibility if only upgrading CLI but not the Library. Then run `amplify codegen models` to regenerate the Model classes. The ModelFieldType `.customType` has been replaced with `embedded(type:)` and `embeddedCollection(of:)`.